starters/graphql-serverless-contentful/.build/package.json

```diff
@@ -0,0 +1 @@
+/Users/markshenouda/Projects/first-serverless/package.json
```

Comment:
I feel like we should remove this line because it mentions your personal user account

Reply:
Oh this `.build` folder shouldn't be here